sklearn/ensemble/tests/test_forest.py

```diff
@@ -197,7 +196,11 @@ def test_probability():
         yield check_probability, name
 
 
-def check_importances(name, criterion, X, y):
+def check_importances(name, criterion, X, y, dtype):
```

Comment:
To make the test reports easier to read, please put dtype before X and y. Or even put X, y as global variables at the beginning of the module.